src/sentry/migrations/0363_mv_user_avatar.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 2.2.28 on 2023-02-25 07:21
+
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0362_break_project_integration_fk"),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            database_operations=[
+                migrations.RunSQL(
+                    """
+                    ALTER TABLE "auth_user" ADD COLUMN "avatar_type" smallint NOT NULL DEFAULT 0;
+                    ALTER TABLE "auth_user" ADD COLUMN "avatar_url" varchar(120) DEFAULT NULL;
+                    """,
+                    reverse_sql="""
+                    ALTER TABLE "auth_user" DROP COLUMN "avatar_url";
+                    ALTER TABLE "auth_user" DROP COLUMN "avatar_type";
+                    """,
+                    hints={"tables": ["auth_user"]},
+                ),
+            ],
+            state_operations=[
+                migrations.AddField(
+                    model_name="user",
+                    name="avatar_type",
+                    field=models.PositiveSmallIntegerField(default=0),
+                ),
+                migrations.AddField(
+                    model_name="user",
+                    name="avatar_url",
+                    field=models.CharField(max_length=120, null=True),
```

Comment:
Should this be longer or are URLs a fixed length?

Reply:
all avatar urls, or at least the ones I looked at, look like: `https://sentry.io/avatar/188f858e711e455aa44f419ec6fb8ba1/`